.gitignore

```diff
@@ -39,3 +39,4 @@ pip-log.txt
 *.sublime-workspace
 *.swp
 *.sw[po]
+data
```

Comment:
:thinking: this is too specific to your current environment, it should not be globally ignored IMO.